var/spack/repos/builtin/packages/libgcrypt/package.py

```diff
@@ -32,8 +32,10 @@ class Libgcrypt(AutotoolsPackage):
        key algorithms, large integer functions, random numbers and a lot
        of supporting functions. """
     homepage = "http://www.gnu.org/software/libgcrypt/"
-    url      = "http://gd.tuwien.ac.at/pub/gnupg/libgcrypt/libgcrypt-1.6.2.tar.bz2"
+    url 	 = "http://gnupg.org/ftp/gcrypt/libgcrypt/libgcrypt-1.7.6.tar.bz2"
```

Comment:
I'm not sure what the consensus is; @adamjstewart would know better.  However, for a small number (4) of packages, we need to use `http`, so that systems without a working curl can use Spack to build a working curl.  Otherwise it's a catch-22.  See #3113